vuepress/docs/.vuepress/config.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
             link: '/about/'
           },
           {
-            text: 'Github',
+            text: 'GitHub',
```

Comment:
This line was the thing that was updating the other `.html` files, but I've dropped the changes to the html files...